bundle/jsky.app.ot/src/main/scala/jsky/app/ot/ags/BagsManager.scala

```diff
@@ -92,20 +92,22 @@ final class BagsManager(executor: ScheduledThreadPoolExecutor) {
    * Atomically enqueue a task that will consider the specified observation for BAGS lookup, after
    * a delay of at least `delay` milliseconds.
    */
-  def enqueue(observation: ISPObservation, delay: Long): Unit =
+  def enqueue(observation: ISPObservation, delay: Long, initialEnqueue: Boolean = false): Unit =
```

Comment:
Ok if `isEligibleForBags` only happens once then I suggest you pass the eligibility check as a parameter.

```scala
def enque(observation: ISPObservation, delay: Long)(continue: ObsContext => Boolean): Unit = {
  ...
  ...  ObsContext.create(obs).asScalaOpt.filter(continue).foreach { ctx => ...
}
```

So the initial call is `enque(..., ...)(_.isEligibleForBags)` and subsequent `enque(..., ...)(_ => true)`.

Whatever you do here please update the doc to indicate what the check does and why it only happens once.